src/test/java/accounting/data/TransactionDAOTest.java

```diff
@@ -1,5 +1,23 @@
 package accounting.data;
 
+import org.junit.Before;
+import org.junit.Test;
+import org.skife.jdbi.v2.DBI;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class TransactionDAOTest {
+    private TransactionDAO dao;
+
+    @Before
+    public void setup() {
+        DBI dbi = new DBI("jdbc:mysql://localhost:3306/accounting", "root", "root");
```

Comment:
Does this mean tests are running against the database? Should we have a test database for this?